runtime/src/runtime/policy.rs

```diff
@@ -339,12 +347,19 @@ pub mod policy_constants {
     pub const WORKER_KEY_CHANGE_DELAY: ChainEpoch = CHAIN_FINALITY;
 
     /// Minimum number of epochs past the current epoch a sector may be set to expire.
-    pub const MIN_SECTOR_EXPIRATION: i64 = 180 * EPOCHS_IN_DAY;
+    pub const MIN_SECTOR_COMMITMENT: i64 = 180 * EPOCHS_IN_DAY;
 
     /// Maximum number of epochs past the current epoch a sector may be set to expire.
     /// The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
     /// and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
-    pub const MAX_SECTOR_EXPIRATION_EXTENSION: i64 = 540 * EPOCHS_IN_DAY;
+    pub const MAX_SECTOR_COMMITMENT_EXTENSION: i64 = 540 * EPOCHS_IN_DAY;
+
+    /// Maximum number of epochs the proof can be valid since activation or latest refresh
+    pub const MAX_PROOF_VALIDITY: i64 = (EPOCHS_IN_YEAR * 3) / 2;
```

Comment:
I generally mean 540 days when I say "a year and a half". Is there a reason to do this one differently?

Reply:
I used the fact that 1.5 * year is a bit longer than 540days to ensure proof expirations line up well.

I can make it explicit by setting the max proof expiration to something like 545 or 550 days.